### PR TITLE
Use query-string instead of qs

### LINKED ISSUE #121: Use query-string instead of qs
Just noticed that history is 60k before minification!

screen shot 2015-10-29 at 6 12 49 pm

### PR DESCRIPTION
**Pull request by mjackson**
Fixes #121

### CODE REVIEW

File: modules/useQueries.js [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
+  return stringify(query).replace(/%20/g, '+')
 }
 
-function defaultParseQueryString(queryString) {
-  return qs.parse(queryString.replace(/\+/g, '%20'))
+const defaultParseQueryString = parse
+
+function isNestedObject(object) {
+  for (var p in object)
+    if (typeof object[p] === 'object')
```
Comment (taion) [inline modules/useQueries.js]: How do you want to handle arrays and `null` here? query-string handles them just fine (though stringified differently from qs in both cases).

File: modules/useQueries.js [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
-  return qs.stringify(query, { arrayFormat: 'brackets' }).replace(/%20/g, '+')
+  return stringify(query).replace(/%20/g, '+')
 }
 
-function defaultParseQueryString(queryString) {
-  return qs.parse(queryString.replace(/\+/g, '%20'))
+const defaultParseQueryString = parse
+
+function isNestedObject(object) {
+  for (var p in object)
```
Comment (taion) [inline modules/useQueries.js]: Nits:
- This should probably be `const`
- `hasOwnProperty` guard? (ESLint can actually be set to warn against that)

File: modules/useQueries.js (hunk 1 above)
Comment (timdorr) [inline modules/useQueries.js]: It looks like it handles those sanely. This is just a developer convenience warning, so they're aware we're not looking to handle nested queries. I wouldn't try to go overboard on supporting a bunch of edge cases here. I think they'll figure it out when they see `%5Bobject+Object%5D` in their URLs :smile:

**Comment by timdorr** [comment]
@mjackson Feel free to smash that commit. I don't know how you like your commit history. I'm more of a multiple-commit kind of dude, but you may want it clean.

**Comment by taion** [comment]
@timdorr Regarding your line comment that got eaten, what I meant was we should probably _not_ warn in those cases, because `query-string` is sane about handling them.

**Comment by timdorr** [comment]
Duh, because arrays are typeof object. I always forget that. Let me amend that commit then.

**Comment by taion** [comment]
Arrays _and_ `null`. I usually just do `p instanceof Object`.

**Comment by timdorr** [comment]
But `[] instanceof Object === true`

**Comment by taion** [comment]
D'oh. Let's squash and merge this then.

**Pull request merged**